include/boost/geometry/algorithms/discrete_hausdorff_distance.hpp

```diff
@@ -262,19 +262,35 @@ inline typename distance_result
             typename point_type<Geometry2>::type,
             Strategy
         >::type
-hausdorff_distance(Geometry1 const& g1, Geometry2 const& g2, Strategy const& strategy)
+hausdorff_distance(Geometry1 const& geometry1, Geometry2 const& geometry2, Strategy const& strategy)
 {
-    return dispatch::hausdorff_distance<Geometry1, Geometry2>::apply(g1, g2, strategy);
+    return dispatch::hausdorff_distance<Geometry1, Geometry2>::apply(geometry1, geometry2, strategy);
 }
 
 // Algorithm overload using default Pt-Pt distance strategy
+/*!
```

Comment:
The line above this comment

    // Algorithm overload using default Pt-Pt distance strategy

is not needed.

Reply:
ok.